tests/e2e/testdata/scenarios/sql/unmanage-edition/script.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# 00: Create a SQLInstance with edition ENTERPRISE_PLUS
+apiVersion: sql.cnrm.cloud.google.com/v1beta1
+kind: SQLInstance
+metadata:
+  name: sqlinstance-${uniqueId}
+spec:
+  databaseVersion: POSTGRES_15
+  region: us-central1
+  settings:
+    tier: db-g1-small
+    edition: ENTERPRISE_PLUS
+  resourceID: sqlinstance-${uniqueId}
+---
+# 01: Simulate a creation in GCP or GCCLOUP
+TEST: ABANDON
+apiVersion: sql.cnrm.cloud.google.com/v1beta1
+kind: SQLInstance
+metadata:
+  name: acquisition-sqlinstance-${uniqueId}
+  annotations:
+    cnrm.cloud.google.com/deletion-policy: abandon
+spec:
+  databaseVersion: POSTGRES_15
+  region: us-central1
+  settings:
+    tier: db-g1-small
+    edition: ENTERPRISE_PLUS
+  resourceID: sqlinstance-${uniqueId}
+---
+# 02: Add the unmanaged annotation and acquire the resource. No change expected.
+apiVersion: sql.cnrm.cloud.google.com/v1beta1
+kind: SQLInstance
+metadata:
+  name: sqlinstance-${uniqueId}
+  annotations:
+    cnrm.cloud.google.com/unmanaged: "spec.settings.edition,spec.settings.edition"
```

Comment:
Will follow up for better code handling (dedup, etc.)

For now, this test shows that repeated values do not break the extraction logic.